sormas-api/src/test/java/de/symeda/sormas/api/doc/DataDictionaryGenerator.java

```diff
@@ -207,6 +216,11 @@ private XSSFSheet createEntitySheet(XSSFWorkbook workbook, Class<? extends Entit
 				fieldValueCell.setCellValue(Boolean.TRUE.toString() + ", " + Boolean.FALSE.toString());
 			}
 
+			//sensitive data
+			XSSFCell dataProtectionCell = row.createCell(EntityColumn.DATA_PROTECTION.ordinal());
+			if (field.getAnnotation(SensitiveData.class) != null)
```

Comment:
@sergiupacurariu You also have to check for a PersonalData annotation. When the PersonalData annotation is used, it should be "personal". Otherwise, when the SensitiveData is used, it should be "senstive". Otherwise it should stay empty.

Reply:
updated